app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +51,35 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
   Host = 'localhost'
   Port = 8200
-  Path = '/v1/secret/edgex/appservice'
+  Path = '/v1/secret/edgex/edgex-appservice/'
   Protocol = 'http'
-  Tokenfile = 'res/token.json'
+  TokenFile = './res/token.json'
+  # Number of attempts to retry retrieving secrets before failing to start the service.
+  AdditionalRetryAttempts = 10
+  # Amount of time to wait before attempting another retry
+  RetryWaitPeriod = "1s"
 
   [SecretStore.Authentication]
-  AuthType = 'X-Vault-Token'
-  # Vault root token that's used for authentication
-  AuthToken = 's.imTIVMl1OWhlX3zMAhSyaFlO'
+    AuthType = 'X-Vault-Token'
+
+# Exclusive Secret Store
+[SecretStoreExclusive]
+  Host = 'localhost'
+  Port = 8200
+  Path = '/v1/secret/edgex/secrets-example/'
+  Protocol = 'http'
```

Comment:
Change to `https`

Reply:
im running Vault in dev mode, which runs the server in HTTP

Reply:
Ah, OK. Please add this as a comment so folks don't copy it and expect it to work with Vault in non-dev mode.